spec/unit/lib/rest_controller/paginated_collection_renderer_spec.rb

```diff
@@ -293,6 +293,44 @@ module VCAP::CloudController::RestController
         end
       end
 
+      context 'when order-by' do
```

Comment:
What happens if one of the `order-by` values is not valid, e.g., `bogus`? Should you just pass it or should we attempt to fail fast soon? 

Also, small mistakes like: `order-by=id, ,name` will result in `['id', ' ', 'name']`. But maybe this is not something that can be prevented.

Anyhow, just curious. My .02